### PR TITLE
[CI:DOCS] Update "CONTRIBUTING" file with Debian/Ubuntu dependencies"

### PR DESCRIPTION
**Pull request by daniloglima**
Signed-off-by: Danilo Lima <danilo.glima@outlook.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,6 +92,13 @@ You need install some dependencies before building a binary.
 
 #### Fedora
 
+  ```shell
+  $ sudo apt-get install -y libsystemd-dev libgpgme-dev libseccomp-dev
+  $ export PKG_CONFIG_PATH="/usr/lib/pkgconfig"
+  ```
+
+#### Debian / Ubuntu
+
```
Comment (giuseppe) [inline CONTRIBUTING.md:101]: It doesn't seem correct.  The previous command must remain under Fedora, while the new one should be under `Debian / Ubuntu`.

**Review by vrothberg — COMMENTED** [review]
LGTM
@siretart @giuseppe PTAL

**Comment by vrothberg** [comment]
Can you rebase to the latest head?  That will pull in fixes to let the tests pass.

**Comment by daniloglima** [comment]
> Can you rebase to the latest head? That will pull in fixes to let the tests pass.

Done

**Review by vrothberg — COMMENTED** [review]
Thanks, LGTM!

@containers/podman-maintainers PTAL

**Pull request merged**